app.rb

```diff
@@ -31,13 +33,16 @@ def create_student(age, name, parent_permission)
     student = Student.new(age, @default_classroom, name, parent_permission: parent_permission)
     @assign_people = @people_controller.all_people
     @assign_people << student
+    puts student
+    write_people(student)
     puts 'Student created successfully!'
   end
 
   def create_teacher(age, name, specialization)
-    teacher = Teacher.new(age, name, specialization)
+    teacher = Teacher.new(age, specialization, name)
     @assign_people = @people_controller.all_people
     @assign_people << teacher
+    write_people(teacher)
```

Comment:
- [ ] **Optional:** The date input method should specify the format in which you expect the user to enter the date.

![image](https://user-images.githubusercontent.com/27709832/175246230-7bb04d04-7f0e-4005-84f9-69438d717928.png)